frontend/packages/ux-editor/src/components/config/editModal/EditDataModelBinding/EditBinding/SelectDataModelBinding.tsx

```diff
@@ -32,23 +43,26 @@ export const SelectDataModelBinding = ({
 
   return shouldDisplayFeature('multipleDataModelsPerTask') ? (
     <FormField
-      id={selectedDataModel}
+      id={id}
       onChange={handleDataModelChange}
-      value={selectedDataModel}
+      value={currentDataModel}
       propertyPath={propertyPath}
       label={t('ux_editor.modal_properties_data_model_binding')}
       renderField={({ fieldProps }) => (
         <StudioNativeSelect
           className={classes.selectDataModel}
           {...fieldProps}
           label={t('ux_editor.modal_properties_data_model_binding')}
-          id={selectedDataModel}
+          id={id}
           onChange={(e) => fieldProps.onChange(e.target.value)}
           size='small'
         >
-          {dataModels.map((element) => (
-            <option key={element} value={element}>
-              {element}
+          <option key={defaultDataTypeName} value={defaultDataTypeName}>
```

Comment:
Here maybe we need to add check or something else to handle the case where there is no datatype 🙂

Reply:
Agreed, will look into this